index.html

```diff
@@ -7166,6 +7166,45 @@ <h2>application/vp+ld+json</h2>
 
     </section>
 
+    <section class="appendix informative">
+      <h2>Additional Diagrams</h2>
+
+      <p id="info-graph-vp-mult-creds-expl">
+        <a href="#info-graph-vp-mult-creds"></a> below is a variant of <a href="#info-graph-vp"></a>:
+        a [=verifiable presentation=] referring to two [=verifiable credentials=], and using <a>embedded proofs</a> based on [[?VC-DATA-INTEGRITY]]. 
+        Each [=verifiable credential graph=] is connected to 
+        its own separate [=proof graph=]; the <code>verifiableCredential</code> property is used
+        to connect the [=verifiable presentation=] to the [=verifiable credential graphs=].
+        The [=presentation=] [=proof graph=] represents the digital signature of the [=verifiable presentation graph=], both [=verifiable credential graphs=], and the [=proof graphs=] linked from the [=verifiable credential graphs=]. The complete [=verifiable presentation=]
+        consists, in this case, of six information [=graphs=].
+      </p>
+
+      <figure id="info-graph-vp-mult-creds">
+        <img style="margin: auto; display: block; width: 100%;" src="diagrams/vp-graph-mult-creds.svg" alt="Diagram with a
+      'verifiable presentation graph' on top connected via a 'proof' to
+      a 'verifiable presentation proof graph on the bottom.  The verifiable
+      presentation graph has and object 'Presentation ABC' with 3 properties: 'type'
+      of value VerifiablePresentation, 'termsOfUse' of value 'Do Not Archive'. The
+      graph is annotated with the parenthetical remark '(the default graph)'. This
+      graph is connected, through 'verifiableCredential', to the part of the figure
+      that consists two variants of Figure 6 (one is identical, the other with
+      minor changes on the labels referring to validity dates, name of the person,
+      and the values for the nonce and the signature values), 
+      except that the verifiable credential graphs are
+      annotated to be nameds graphs instead of a default graph.
+      The verifiable presentation proof graph has and object with 'Signature 8920'
+      with 5 properties: 'type' of DataIntegrityProof, 'verificationMethod' of Example
+      Presenter Public Key 11, 'created' of 2024-01-02T12:43:56Z, a 'nonce' of 'hasdkyruod87j', 
+      and 'proofValue' of 'zpewJHoan87='. This graph is annotated with the parenthetical remark '(a
+      named graph)'">
+        <figcaption style="text-align: center;">
+          A variant of <a href="#info-graph-vp"></a>: information [=graphs=] associated with a [=verifiable presentation=] referring to two 
+          verifiable credentials, using an [=embedded proof=] based on [[[VC-DATA-INTEGRITY]]] [[?VC-DATA-INTEGRITY]].
+        </figcaption>
+      </figure>
```

Comment:
It looks like GitHub can't properly display the changes I'm suggesting. Please review by copying and pasting into your diff tool, as there are a number of changes to punctuation as well as text.
```suggestion
        <img style="margin: auto; display: block; width: 100%;" src="diagrams/vp-graph-mult-creds.svg" alt="Diagram with a
      'verifiable presentation graph' on top, connected via a 'proof' to
      a 'verifiable presentation proof graph' on the bottom. The verifiable
      presentation graph has an object, 'Presentation ABC', with 3 properties: 'type'
      with value 'VerifiablePresentation'; 'termsOfUse' with value 'Do Not Archive';
      and two instances of 'verifiableCredential', detailed below. This
      graph is annotated with a parenthetical remark, '(the default graph)'. This
      graph is connected, through 'verifiableCredential', to the part of the figure
      that consists two variants of Figure 6 (one is identical; the other has
      minor differences in the labels referring to validity dates, the name of the person,
      and the values for the nonce and the signature),
      except that these verifiable credential graphs are
      annotated to be named graphs instead of a default graph.
      The verifiable presentation proof graph has an object labeled 'Signature 8920'
      with 5 properties: 'type' with value 'DataIntegrityProof'; 'verificationMethod' with value 'Example
      Presenter Public Key 11'; 'created' with value '2024-01-02T12:43:56Z'; 'nonce' with value 'hasdkyruod87j';
      and 'proofValue' with value 'zpewJHoan87='. This graph is annotated with the parenthetical remark '(a
      named graph)'">
        <figcaption style="text-align: center;">
          A variant of <a href="#info-graph-vp"></a>: information [=graphs=] associated with a [=verifiable presentation=] referring to two
          verifiable credentials, using an [=embedded proof=] based on [[[VC-DATA-INTEGRITY]]] [[?VC-DATA-INTEGRITY]].
        </figcaption>
      </figure>
```

Reply:
I do not know what happened with github... Anyway, I have manually changed the part you edited to hold the new version.